lib/useOnyx.ts

```diff
@@ -40,7 +40,7 @@ type UseOnyxSelectorOption<TKey extends OnyxKey, TReturnValue> = {
      * when the subset of data changes. Otherwise, any change of data on any property would normally
      * cause the component to re-render (and that can be expensive from a performance standpoint).
      */
-    selector?: Selector<TKey, unknown, TReturnValue>;
+    selector?: UseOnyxSelector<TKey, TReturnValue>;
```

Comment:
Could you update the `selector` JSDoc to account for this change?

Reply:
Sure thing! Could you tell a little bit more which fragment should be updated? Thanks!

Reply:
Actually nvm, the description still makes sense with the change 😅